src/test/rustdoc/favicon-path.rs

```diff
@@ -0,0 +1,6 @@
+// compile-flags:-Z unstable-options --favicon-path ./src/librustdoc/html/static/favicon.ico
```

Comment:
Loading the file like this won't work on travis, since `x.py` is called from a different directory there - this path isn't valid.

Reply:
Well, I'll just create or use another file. :)